lmcache/cache_engine.py

```diff
@@ -185,11 +185,15 @@ def _make_chunks_skip_existing(
         tokens: torch.Tensor,
         kv_tensors: torch.Tensor,
         fmt: str,
+        num_skip_chunk=0,
```

Comment:
The "physical meaning" of `num_skip_chunk` does not match its name. IIUC, the physical meaning here is "number of prefix chunks to skip". Can we have a better variable name?
@YaoJiayi feel free to chime in if you have any thoughts on this.